### PR TITLE
Lesson 2.3 technical review

### PR DESCRIPTION
**Pull request by andrewk**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -56,7 +57,7 @@ Bad:
 render() {
   return (
     <div>
-      { window.loggedIn? ? <SecretComponent /> : null }
```
Comment (andrewk) [inline readme.md:59]: Invalid property name. Causes a syntax error on node 4.

File: readme.md [hunk 2]
```diff
@@ -64,15 +65,18 @@ render() {
 
 Good:
 ```jsx
-render() {
-  let secretComponent
```
Comment (andrewk) [inline readme.md:68]: This leads to `GodFunction` anti-pattern.

File: readme.md (hunk 1 above)
Comment (dphaener) [inline readme.md:59]: Heh whoops. Looks like a bit of Ruby syntax slipped in.

File: readme.md (hunk 2 above)
Comment (dphaener) [inline readme.md:68]: :+1:

**Pull request merged**